proposals/extensions.md

```diff
@@ -693,7 +693,8 @@ TODO4 need to merge extension members and extension methods
     - Merge the results
     - Next, members that are hidden by other members are removed from the set.  
       (note: "base types" means "base extensions and underlying type" for extension types)
-    - Finally, having removed hidden members:
+    - Next, extension members are removed if any more specific extension member is applicable.
```

Comment:
>Next, extension members are removed if any more specific extension member is applicable. [](http://example.com/codeflow?start=6&length=88)

This is very vague. What does it mean to be more specific should be clearly specified. Perhaps in a separate section. I think our goal at this point should be to clearly describe what the implementation is doing, so that the rules could be reviewed/discussed/approved. Like less specific methods are not removed by more specific methods, etc.